src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/payment/processor/FunctionalQueueConfig.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor;
+
+import com.microsoft.azure.servicebus.IMessageReceiver;
+import com.microsoft.azure.servicebus.QueueClient;
+import com.microsoft.azure.servicebus.ReceiveMode;
+import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Profile;
+
+import static org.mockito.Mockito.mock;
+
+public class FunctionalQueueConfig {
+
+    @Value("${azure.servicebus.payments.write-connection-string}")
+    private String paymentsQueueWriteConnectionString;
+
+    @Bean("payments")
+    @Profile("nosb")
```

Comment:
if `nosb` stands for 'no servicebus', why are we configuring real client to a queue here?

Reply:
Agree, 'nosb' is a bit misleading here, 'functest' or smth like this should be better, but will cause a lot of changes in prod code where !nosb is used